lib/tasks/demo.rake

```diff
@@ -379,9 +404,13 @@ namespace :demo do
       patients << patient
     end
 
+    data[:num_unique_patients] = (Patient.where(purged: false).size * 0.3).to_i
```

Comment:
What is the reason for doing this?

Reply:
this is just part of the refactor, the idea is still the same in that when generating patients via `demo:populate`, all those patients should be unique but when generating purged patients, we can generate batches of duplicate patients much faster

Reply:
gotcha